src/app/child-dev-project/children/children.service.ts

```diff
@@ -93,15 +93,15 @@ export class ChildrenService {
   queryLatestRelation(
     childId: string
   ): Promise<ChildSchoolRelation | undefined> {
-    return this.queryRelationsOf("child", childId, true).then((relations) =>
+    return this.queryRelationsOf("child", childId).then((relations) =>
       relations.length > 0 ? relations[0] : undefined
     );
   }
 
   async queryRelationsOf(
     queryType: "child" | "school",
     id: string,
-    onlyActive = false
+    onlyActive = true
```

Comment:
wouldn't it make slightly more sense to call this `includeInactive` instead and swap the logic? Sorry for digging at this again and again :see_no_evil:

Reply:
Extracted another function as this boolean parameter was kind of confusing anyways